packages/dialtone-vue3/prototypes/dialpad/leftbar/components/options_dropdown.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <dt-dropdown
+    navigation-type="arrow-keys"
+    placement="bottom-start"
+    @opened="openDropdown"
+  >
+    <template #anchor="{ attrs }">
+      <dt-button
+        ref="dropdownAnchor"
+        v-dt-tooltip="'Section options'"
+        class="d-fc-tertiary"
+        importance="clear"
+        kind="muted"
+        size="sm"
+        circle
+        aria-label="Section Options"
+        v-bind="attrs"
+        @click.stop="openDropdown"
+      >
+        <template #icon="{ iconSize }">
+          <dt-icon
+            name="more-vertical"
+            :size="iconSize"
+          />
+        </template>
+      </dt-button>
+    </template>
+
+    <template #list="{ close }">
+      <template v-for="option in menuOptions">
+        <dt-dropdown-separator
+          v-if="option.separator"
+          :key="`${option.value}separator`"
+        />
+        <dt-list-item
+          v-else
+          :key="option.value"
+          navigation-type="arrow-keys"
+          role="menuitem"
+          @click="selectMenuOption(option, close);"
+        >
+          {{ option.name }}
+        </dt-list-item>
+      </template>
+    </template>
+  </dt-dropdown>
+</template>
+
+<script setup>
+import { DtDropdown, DtDropdownSeparator } from '@/components/dropdown';
+import { DtListItem } from '@/components/list_item';
+import { DtButton } from '@/components/button';
+import { DtIcon } from '@/components/icon';
+import { ref, defineEmits } from 'vue';
+
+const emits = defineEmits(['select-menu-option', 'open-dropdown']);
+
+const menuOptions = ref([
+  { value: 0, name: 'Mark all as read' },
+  { value: 1, separator: true },
+  { value: 2, name: 'Create new channel' },
+  { value: 3, name: 'Browse channels' },
+]);
+
+const openDropdown = (value) => {
+  emits('open-dropdown', value);
+};
+
+const selectMenuOption = (option, close) => {
+  close?.();
+  emits('select-menu-option', option);
```

Comment:
Regarding events through the component,
We are emitting them but not showing it in the `Actions` tab of storybook, also we are not using it at all, Im guessing we can remove them or keep it track in the `Actions` tab.

Reply:
Since it is a prototype I guess we can keep it track in the `Actions` tab to let us know which events are being fired through the usage.

Not so sure, open to discussion, it's fine if we leave it as it is now.

Reply:
Hmm probably okay to not emit them in a prototype in an internal component like this.

Reply:
Yes, that's what I was thinking, we don't really need to emit the events in this component. I'll left the component as is right now.